articles/aks/security-bulletins/overview.md

```diff
@@ -22,6 +22,29 @@ These updates cover security information related to the following AKS components
 - Azure Kubernetes Service Node Image (AKS Node Image)
 - Azure Kubernetes Service Addons (AKS add-ons)
 
+---
+## AKS-2025-009 Important Security Update for Calico Users
+
+**Published Date**: June 25, 2025
+
+### Description
+This bulletin provides an update on the security patching model for Calico in Azure Kubernetes Service (AKS). AKS-managed Calico is now fully aligned with upstream [Calico releases](https://github.com/projectcalico/calico/releases). This means that AKS will no longer independently patch Calico images but will instead mirror upstream Calico builds directly. 
+
+As a result, CVEs affecting Calico will remain unpatched in AKS until a fix is available upstream. This change ensures consistency with upstream behavior and improves transparency in patch timelines.
+
+### References
```

Comment:
Assume its not worth it/bad to mention we don't do this because of the complexitity of some of calico nodes image builds. I've been pretty transparent tigera about this so I don't think they would take offense but assume we don't normally give reasons.